src/BCrypt/BCrypt+AlgorithmIdentifiers.cs

```diff
@@ -6,7 +6,11 @@ namespace PInvoke
     /// <content>
     /// Contains the <see cref="AlgorithmIdentifiers"/> nested class.
     /// </content>
-    public partial class BCrypt
```

Comment:
Isn't it confusing for customer that would want to import static both classes (Don't know if such consumer are realistic)